### PR TITLE
Document heartbeat topic

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

File: app/routes/docs.faq.md [hunk 1]
```diff
@@ -58,6 +58,23 @@ Clients connecting to GCN only need to be able to make _outbound_ (egress) TCP c
   </tbody>
 </table>
 
+### How can I tell that my Kafka client is working?
+
+Subscribe to the topic `gcn.heartbeat`, which broadcasts a test message approximately once a second. The contents of the heartbeat messages looks like this:
+
+```json
+{
+  "$schema": "https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices/core/Alert.schema.json",
```
Comment (dakota002) [inline app/routes/docs.faq.md:67]: Should we use the `.../schema/stable/gcn...` route here?

Comment (lpsinger) [reply app/routes/docs.faq.md:67]: Although, we _could_ generate that URL automatically from the @nasa-gcn/schema dependency... one moment.

**Review by dakota002 — APPROVED** [review]

File: app/routes/docs.faq.md (hunk 1 above)
Comment (dakota002) [reply app/routes/docs.faq.md:67]: Gotcha, I didn't see your reply before my edit, but overall this all looks good

Comment (lpsinger) [reply app/routes/docs.faq.md]: Take a look again please.

**Review by dakota002 — APPROVED** [review]

**Pull request merged**